SmartDeviceLinkTests/UtilitiesSpecs/HTTP Connection/SDLURLRequestTaskSpec.m

```diff
@@ -1,140 +0,0 @@
-#import <Quick/Quick.h>
-#import <Nimble/Nimble.h>
-#import <OHHTTPStubs/OHHTTPStubs.h>
-
-#import "SDLURLRequestTask.h"
-
-
-QuickSpecBegin(SDLURLRequestTaskSpec)
```

Comment:
We probably need replacement tests of some sort to make sure that the URL requests are set up properly? We don't need to test actual functionality, I don't think.